src/main/kotlin/no/nav/helsearbeidsgiver/arbeidsgivernotifikasjon/ArbeidsgiverNotifikasjonKlient.kt

```diff
@@ -252,48 +186,6 @@ class ArbeidsgiverNotifikasjonKlient(
         loggInfo("Slettet (hard) sak med id '$id'.")
     }
 
-    suspend fun whoami(): String? =
```

Comment:
Ikke noen big deal for meg, men den whoami var litt kjekk for å bare teste / verifisere kobling, uten at man trenger noen sak / oppgave med tilhørende tilganger. Så selv om den ikke er i bruk per se, kan den være praktisk.

Reply:
Aiit, jeg kan legge den tilbake igjen. Vi trenger kanskje ikke testen?